### ISSUE TITLE
ETH: Implement EIP-1898 support to ETH RPC

### ISSUE THREAD

**Issue by adlrocha** [consensus-shipyard/fendermint#183]
Lotus recently introduced as part of its latest version compatibility fo (EIP-1898)[https://github.com/filecoin-project/lotus/pull/10815], apparently some applications like The Graph need this for their operation. We should verify if we already implement this support and if not include it in Fendermint's ETH RPC.

**Comment by aakoshh** [comment]
I don't think the `bug` label is justified here, it's a new feature added to Ethereum, not supported by the `ethers` library we use, so it's not working in an incorrect way. 

If we have to support it it's because a tool drops support for legacy formats, but because of Tendermint's forward only nature, the "is canonical" flag doesn't make any difference in our case.

**Comment by adlrocha** [comment]
> I don't think the bug label is justified here

I completely agree :) Sorry about that, I wanted to include it to the `IPC bug tracker` to have it tracked there and it may have assigned that label automatically, and I didn't realized this happened. I removed it.

**Comment by aakoshh** [comment]
Thanks, I realised later that's what happened when I tried to create a new issue and it was automatically a bug because that's the only template we have :smiley:
